README.md

```diff
@@ -1,39 +1,82 @@
+   1. We want the FHIR "R4" (current) version. If a particular questionnaire is only available as an earlier version of FHIR (eg DSTU2, or STU3), we'll probably want to start there and adapt it to the R4 format.
+   1. If we've already have a non-FHIR implementation of the questionnaire in one of our systems (eg dhair2, DCW), compare verbiage, scoring, with what's found. We generally need to retain whatever verbiage we've been using historically, but confirm on a case-by-case basis w/ research staff.
+1. If we don't find an existing FHIR Questionnaire, here are the alternatives:
+   1. Copy existing similar qnr from our repo
+      1. Example: CIRG-PainTracker-STOP (very brief qnr, 4 yes/no questions)
+   1. Create one de novo w/ help from a script (see chatgpt script) - makes sense for something like PainTracker TRT - fairly long, all codings novel.
+      1. Example: CIRG-PainTracker-TRT
 
 # Fields and how we use them
 - "id"
    - usage: this is how we refer to the questionnaire in code, CarePlans, etc.
-   - REQUIRED.
-   - example: "CIRG-PHQ9"
+   - REQUIRED 1..1 (same as standard), and we need it to be a known & reliable value (not assigned by the FHIR server) - that part is not per standard.
+   - QuestionnaireResponse.questionnaire points at this via eg "Questionnaire/CIRG-PHQ-4"
+   - example: "CIRG-PHQ-4"
+   - Our profile: add the above info
 - "title"
-   - usage: human readable.
-   - REQUIRED.
+   - usage: human friendly.
+   - REQUIRED (1..1). THIS IS A CHANGE FROM THE STANDARD, which says 0..1. Add this to our profile.
    - example: "Everyday Cognition - Participant Self Report Form [ECog]"
+   - In our [questionnaire filler](https://github.com/uwcirg/asbi-screening-app), this is used for the page's <head><title>.
 - "description"
-   - usage: A sentence or two description of the questionnaire. This may be presented in the questionnaire user interface on a page prior to the questionnaire, and/or included in reports. 
-   - REQUIRED.
+   = REQUIRED (1..1). This is in contrast to spec (0..1).
+   - Per spec: "Natural language description of the questionnaire"; [Markdown w/out HTML](https://build.fhir.org/datatypes.html#markdown).
+   - 2023-10-12 Moving forward we'll use this for descriptive text to displayed to non-lay users (eg in the summary report). Also, while we'll use Markdown, we expect that the front-end may convert that to html (and indeed we'll include e.g. links to references here).
+   - Contrast this with item[0]._text.extension (see below).
 - "code"
    - usage: uniquely identify the questionnaire according to some system, eg at LOINC. Not our canonical reference (see "id"). See https://github.com/uwcirg/fhir-questionnaires/pull/2/files#r974579864
 - "name"
-   - usage: arbitrary?
-- "linkId"
-   - usage: QuestionnaireResponse will refer to this
-   - REQUIRED.
+   - Standard says 0..1; computer friendly.
+   - Do we use this anywhere Amy? I see that it's populated for the DCW Questionnaires.
```

Comment:
@mcjustin `name` element, along with the `id` element is used by our ASBI app to determine what questionnaire(s) to query for.  See [here](https://github.com/uwcirg/asbi-screening-app/blob/9b4d6daa9b31bc799af1c69de612f8b495fd3e44/src/util/screening-selector.js#L211)

Reply:
thanks, I've just edited to reflect this.